components/Projects/index.js

```diff
@@ -29,8 +28,8 @@ export default function Projects({ projects, favourites, onToggleFavourite }) {
   return (
     <StyledList>
       {projects.map((project) => (
-        <ProjectCard key={project.slug}>
-          <Link href={`/projects/${project.slug}`}>
+        <ProjectCard key={project._id}>
+          <Link href={`/${project._id}`}>
```

Comment:
having the dynamic route directly in the root can cause problems and does not really adds a value to the UX. I would suggest to move the dynamic route into the "projects" route.

Reply:
We do not see where we could add the Link instead to route to our details page, so for now we left as is.
Maybe you could give us more input directly.